solutions/deployment/scripts/init-auth.py

```diff
@@ -8,15 +8,15 @@
 
 client = boto3.client('cognito-idp', region_name=sys.argv[2])
 '''
+python3 init-auth.py <ClientId> <Region> <USERNAME> <PASSWORD>
 example run:
-python3 init-auth.py <ClientId> <Region>
-python3 init-auth.py 12pgvi3gsl32qp9h8lg130arr0 us-west-2
+python3 init-auth.py 12pgvi3gsl32qp9h8lg130arr0 us-west-2 workshopuser Master123!
```

Comment:
Should this part be removed? `workshopuser Master123!`

Reply:
```suggestion
python3 init-auth.py 12pgvi3gsl32qp9h8lg130arr0 us-west-2 example_user example_password
```